src/tests/CampaignAdSchedules.test.ts

```diff
@@ -46,6 +48,48 @@ describe('Campaign Ad Schedules', async () => {
         criterion_id = new_criterion.id
         done()
     })
+    
+    it('Creates 2 New Campaign Ad Schedules', async done => {
+        expect.assertions(1)
```

Comment:
We expect more than one assertion here, I spotted this in most of added tests. 
We need to change that otherwise Jest will ignore all assertions after the first one.

Reply:
These should be fixed now